app/views/posts/index.html.erb

```diff
@@ -1 +1,46 @@
-<p>All Posts goes here</p>
+
+
+  <div class="container">
+    <div class="card" style="width: 100%">
+      <ul class="list-group list-group-flush">
+        <li class="list-group-item ">
+            <div class="d-flex justify-content-between">
+              <div class="h5"><b>Name: </b><%=@user.name%></div>
+              <p class="mx-4" style="position: relative">
+                Posts:
+                <span
+                  style="
+                    position: absolute;
+                    width: 20px;
+                    height: 20px;
+                    border-radius: 50%;
+                    background: blue;
+                    color: white;
+                    font-size: 12px;
+                    text-align: center;
+                  "
+                  class=""
+                  ><%= @user.posts_counter %></span
+                >
+              </p>
+            </div>
+          </li>
+          <h6 class="px-2 mt-2">Posts</h6>
+        <% @user.posts.each do |post|%>
+        <li class="list-group-item ">
+          <div class="d-flex justify-content-between">
+            <div class="post-container">
+                <h6 class="mb-1"><b>Title: </b><%=post.title%></h6>
+                <p>Comment: <%=post.text%></p>
+            </div>
+          <small>comments: </b><%=post.comments_counter%></b> </small>
+          </div>
+          <div class="">
+            <%= link_to 'Read more', user_post_path(@user, post.id), class: 'btn-sm btn btn-primary' %>
+          <%= link_to 'Back', user_path(@user), class: 'btn-sm btn btn-default' %>
+           </div>
+        </li>
+        <%end%>
+      </ul>
+    </div>
```

Comment:
- [ ] For the user post `index` page please use the `fetch_recent_comments` method to render the most recent 5 comments as shown in the given [view](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_all_posts.png).